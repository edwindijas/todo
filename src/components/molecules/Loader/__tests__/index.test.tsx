import { render } from '@testing-library/react'
import PageLoader from '..'

it ('renders snapshot tests correctly', () => {
  const {container} = render(<PageLoader />)
  expect(container).toMatchSnapshot()
})