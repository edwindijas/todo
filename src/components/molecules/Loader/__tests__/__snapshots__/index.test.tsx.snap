// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot tests correctly 1`] = `<div />`;
